src/components/app/Root.js

```diff
@@ -128,6 +128,8 @@ class ProfileViewWhenReadyImpl extends PureComponent<ProfileViewProps> {
         // with multiple profiles. Only show the ZipFileViewer if the data loaded is a
         // Zip file, and there is no stored path into the zip file.
         return hasZipFile ? <ZipFileViewer /> : <ProfileViewer />;
+      case 'TRANSITIONING_FROM_STALE_PROFILE':
+        return null;
```

Comment:
It's not an action, it's a phase. It's from the app view reducer, and gets triggered with the action `SANITIZED_PROFILE_PUBLISHED`. This action is part of the unfortunate synchronous double dispatch of `SANITIZED_PROFILE_PUBLISHED` and `VIEW_PROFILE`. It will re-render the components here, but shouldn't be displayed to the end user.